packages/paste-website/src/pages/icon-system/how-to-add-an-icon.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: How to add an icon
+description: Guidelines on how to create and export SVG icons for Paste.
+---
+
+import {graphql} from 'gatsby';
+import {Link} from 'gatsby';
+import Img from 'gatsby-image';
+import {Box} from '@twilio-paste/box';
+import {Text} from '@twilio-paste/text';
+import {Button} from '@twilio-paste/button';
+import {PlusIcon} from '@twilio-paste/icons/esm/PlusIcon';
+import {LoadingIcon} from '@twilio-paste/icons/esm/LoadingIcon';
+import {Callout, CalloutTitle, CalloutText} from '../../components/callout';
+import {Grid} from '../../components/grid';
+
+export const pageQuery = graphql`
+  {
+    mdx(fields: {slug: {eq: "/icon-system/how-to-add-an-icon/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<contentwrapper>
+<TableOfContents headings={props.data.mdx.headings} />
+<content>
+
+<small>Written for Sketch 59.1</small>
```

Comment:
This can just be normal-sized body text underneath the h1

Reply:
h1 tags have margin bottom which makes this look weird and misaligned, which is why I cheated by putting this above.  I'll make the change but double check it please and let me know if it looks fine.

Reply:
before 
![image](https://user-images.githubusercontent.com/1592327/69172612-cba96c00-0ac3-11ea-8ca4-2d06f4a535da.png)

After
![image](https://user-images.githubusercontent.com/1592327/69172626-cfd58980-0ac3-11ea-8a23-710fcada2212.png)

Reply:
lgtm!